### PR TITLE
kafka 0.8.2.2

### PR DESCRIPTION
**Pull request by esamson**

### CODE REVIEW

**Comment by DomT4** [comment]
I suspect you need a `ENV["_JAVA_OPTIONS"] = "-Duser.home=#{testpath}"` line at the top of `def install` here.

File: Library/Formula/kafka.rb [hunk 1]
```diff
@@ -26,6 +26,8 @@ def pour_bottle?
   end
 
   def install
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{testpath}"
```
Comment (dunn) [inline Library/Formula/kafka.rb]: Oops; in the install block it should be `buildpath` rather than `testpath`.

Comment (DomT4) [inline Library/Formula/kafka.rb]: Oops. That was my bad advice, apologies @esamson!

**Comment by DomT4** [comment]
@BrewTestBot test this please

File: Library/Formula/kafka.rb [hunk 2]
```diff
@@ -26,6 +26,8 @@ def pour_bottle?
   end
 
   def install
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{buildpath}"
```
Comment (mikemcquaid) [inline Library/Formula/kafka.rb]: @DomT4 I'm wondering if we should just set this globally in all formulae?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

File: Library/Formula/kafka.rb (hunk 2 above)
Comment (DomT4) [inline Library/Formula/kafka.rb]: Still waiting on the problem we discussed in the other thread weeks ago and Tim reported upstream :cry:

Comment (mikemcquaid) [inline Library/Formula/kafka.rb]: Sorry, not sure if I saw that. Could you link me when you get a chance?

Comment (DomT4) [inline Library/Formula/kafka.rb]: https://github.com/Homebrew/homebrew/pull/44567
https://issues.apache.org/jira/browse/MJAVADOC-435

Comment (mikemcquaid) [inline Library/Formula/kafka.rb]: Bah. Cheers!

**Pull request closed without merging**